RocketControlUnitGUI/src/routes/+page.svelte

```diff
@@ -79,9 +79,46 @@
 
 	let containerElement: any;
 
+	interface Timestamps {
+		[key: string]: number;
+	}
+
+	let timestamps: Timestamps = {
+		relay_status: Date.now(),
+		combustion_control_status: Date.now(),
+		rcu_temp: Date.now(),
+		pad_box_status: Date.now(),
+		battery: Date.now(),
+		dmb_pressure: Date.now(),
+		launch_rail_load_cell: Date.now(),
+		nos_load_cell: Date.now(),
+		pbb_pressure: Date.now(),
+		pbb_temperature: Date.now(),
+		rcu_pressure: Date.now(),
+		sob_temperature: Date.now(),
+		sys_state: Date.now()
+	};
+
 	onMount(() => {
 		containerElement = document.querySelector('.container') as HTMLElement;
 
+		setInterval(() => {
+			for (let variable in timestamps) {
+				let elements = document.getElementsByClassName(variable);
+				if (!elements.length) continue;
+
+				for(let i = 0; i < elements.length; i++) {
+					let element = elements[i];
+
+					if (Date.now() - timestamps[variable] > 5000) {
+						element.classList.add('outdated');
+					} else {
+						element.classList.remove('outdated');
+					}
+				}
```

Comment:
Is the `{relayStatusOutdated ? 'outdated' : ''}` doing something else than the `.add` and `.remove` here?

Searchin the whole doc 13 times a second seems kinda expensive, so we might wanna make the delay here bigger, we can check the browser cpu usage on the soar laptop though